src/fullsignalk.js

```diff
@@ -184,10 +184,23 @@ function addValues(context, contextPath, source, timestamp, pathValues) {
 }
 
 function addValue(context, contextPath, source, timestamp, pathValue) {
-  if (_.isUndefined(pathValue.path) || _.isUndefined(pathValue.value)) {
-    console.error("Illegal value in delta:" + JSON.stringify(pathValue));
+  let errMessage = ""
```

Comment:
For some reason I dislike  string concatenation - probably because the ternary makes it a bit hard to comprehend. Imho a better alternative is to use an array: push the error messages to it, then check for length  and `` `...${errs.join(' and ')}...` `` to combine the messages into a readable error message.